mypy.ini

```diff
@@ -1,4 +1,14 @@
 [mypy]
 python_version = 3.8
-ignore_missing_imports = True
 plugins = numpy.typing.mypy_plugin
+
+; third-party packages
+ignore_missing_imports = True
```

Comment:
is there necessary outside of pre-commit?

Reply:
Actually might not need this after we add the types

Reply:
nvm, there are lots of packages missing types, will have to do another PR in future to add more types packages

Reply:
kk, but would like to add those soon. that's free coverage